php/class-plugin.php

```diff
@@ -304,7 +304,7 @@ public function process_post_query_vars( $post_query_vars ) {
 		);
 
 		// White list allowed meta query compare values.
-		$allowed_meta_query_compare_values = array( '=', '!=', '>', '>=', '<', '<=' );
+		$allowed_meta_query_compare_values = array( '=', '!=', '>', '>=', '<', '<=', 'LIKE' );
```

Comment:
@sayedwp oh, you're right 😊 

I suppose then that adding a `customize_object_selector_allowed_meta_query_compare_values` filter would then do the trick, though this is probably overkill and it would be better to just add `LIKE` to `$allowed_meta_query_compare_values` like you had originally!